db/migrations/20170821164207_add_name_to_movies.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.schema.table('movies', (table) => {
+    table.text('name');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ALTER COLUMN title DROP NOT NULL');
+  });
+};
+
+exports.down = function (Knex, Promise) {
+  return Knex.schema.table('movies', (table) => {
+    table.dropColumn('name');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ALTER COLUMN title SET NOT NULL');
```

Comment:
Also worth noting that trying to add the `NOT NULL` constraint is probably a bad idea in production... If you need to rollback and any null `title` entries have been written, the rollback will fail.

Also, just to point it out for the future: http://knexjs.org/#Schema-alter. (Not sure if `lob-api` is using a sufficiently recent version of Knex to take advantage of these, but `woody` is for sure.)